rust/Cargo.toml

```diff
@@ -29,7 +29,7 @@ lazy_static = "1"
 
 # Azure
 reqwest = { version = "0", optional = true }
-azure_core = { git = "https://github.com/Azure/azure-sdk-for-rust", optional = true, rev = "536da42ebefd411feff8ba6a0965865e2741267e" }
+azure_core = { git = "https://github.com/Azure/azure-sdk-for-rust", optional = true, rev =  "536da42ebefd411feff8ba6a0965865e2741267e" }
```

Comment:
nitpick, could you help remove the extra space here to help avoid potential merge conflicts?

Reply:
Oops... didn't notice that.